src/main/java/com/amazon/aocagent/enums/TestCase.java

```diff
@@ -28,7 +29,7 @@
 
   EC2_WIN_TEST(
```

Comment:
what about change the name to "EC2_SSM_TEST" so we can expand it to linux in the future